### PR TITLE
DM-26874: rename pipetask2 to pipetask, remove old pipetask

### PR DESCRIPTION
**Pull request by n8pease**

### CODE REVIEW

File: bin/run_demo.sh [hunk 1]
```diff
@@ -52,8 +52,8 @@ fi
 # can not be the same.
 # Do not specify a number of processors (-j) to test that the default value
 # works.
-pipetask run -d "exposure=903342 AND detector=10" -b DATA_REPO/butler.yaml \
-    -i HSC/calib,HSC/raw/all,refcats --longlog \
+pipetask --long-log run -d "exposure=903342 AND detector=10" -b DATA_REPO/butler.yaml \
```
Comment (andy-slac) [inline bin/run_demo.sh:55]: hmm, I thought we are keeping all options to be per-sub-command options?

**Review by andy-slac — APPROVED** [review]
OK, but check my question.

File: bin/run_demo.sh (hunk 1 above)
Comment (n8pease) [reply bin/run_demo.sh:55]: logging was an exception to this.

Comment (andy-slac) [reply bin/run_demo.sh:55]: I'm fine with that as long as there is a consensus. Initially `pipetask` had global options too, I'm glad we are going in circles 🙂

**Pull request merged**